spec/restricted_input_spec.rb

```diff
@@ -15,7 +15,9 @@
 
     it 'only allows digits' do
       input = find '#credit-card-number'
-      input.send_keys 'abcdefghhijklmnopqrstuvwxyz !@#$%^&*()_=+ 1234567890123456'
+      input.send_keys 'abc'
```

Comment:
hey, if  it gets functional testing to work just leave it. I'm impressed you found that this made it work in the first place